src/data_provider/src/packages/pkgWrapper.h

```diff
@@ -143,19 +145,23 @@ class PKGWrapper final : public IPackageWrapper
                         else if (line == "<key>CFBundleShortVersionString</key>" &&
                                  std::getline(data, line))
                         {
-                            m_version = getValueFnc(line);
+                            auto version = getValueFnc(line);
+                            m_version = version.empty() ? UNKNOWN_VALUE : version;
                         }
                         else if (line == "<key>LSApplicationCategoryType</key>" &&
                                  std::getline(data, line))
                         {
-                            m_groups = getValueFnc(line);
+                            auto groups = getValueFnc(line);
+                            m_groups = groups.empty() ? UNKNOWN_VALUE : groups;
                         }
                         else if (line == "<key>CFBundleIdentifier</key>" &&
                                  std::getline(data, line))
                         {
-                            m_description = getValueFnc(line);
+                            auto description = getValueFnc(line);
+                            m_description = description.empty() ? UNKNOWN_VALUE : description;
 
                             std::string vendor;
+                            m_vendor = UNKNOWN_VALUE;
```

Comment:
"m_vendor" already has UNKNOWN_VALUE as default value. It does not need to reassign the same value.